Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -350,12 +374,13 @@ void LoadBankFromDiskTask::run() {
   std::vector<uint64_t> event_index;
 
   // Open the file
-  ::NeXus::File file(m_loader.alg->m_filename);
+  ::NeXus::File file(m_loader.m_loadAlgorithm->m_filename);
   try {
     // Navigate into the file
-    file.openGroup(m_loader.alg->m_top_entry_name, "NXentry");
+    file.openGroup(m_loader.m_loadAlgorithm->m_top_entry_name, "NXentry");
     // Open the bankN_event group
-    file.openGroup(entry_name, entry_type);
+    if (NexusDescriptor::findAndOpenParentGroup(file, entry_type, entry_name))
+      file.openGroup(entry_name, entry_type);
```

Comment:
Is this still safe? If we can't open the parent group, we can still safely continue?

Reply:
This could be quite slow for files with many entries as `getEntries()` is slow. Can the caller not verify the file looks like an expected file structure and pass this in. For SNS/ISIS we just hardcode the path in the file. Is the ESS structure more fluid?